test-it/ViaTests/ByteArrayString.via

```diff
@@ -0,0 +1,59 @@
+//define (true -43 'shut up about moon men')
+
+define(HelloWorld dv(.VirtualInstrument (
+    Locals:
+    c(
+        e(dv(a(.UInt8 *) (65 66 67 97 98 99)) ba1)
+        e(dv(.Int32 0) encoding)
+        e(dv(.String) s1)
+        e(dv(a(.UInt8 *)) ba2)
+        e(.ErrorCluster err)
+        e(dv(c(
+            e(.Boolean status)
+            e(.Int32 code)
+            e(.String source)
+        ) (false 0 '')) noError)
+
+        e(dv(a(.UInt8 *) (192)) babad_in1)
+        e(dv(a(.UInt8 *) (193)) babad_in2)
+        e(dv(a(.UInt8 *) (245)) babad_in3)
+        e(dv(a(.UInt8 *) (246)) babad_in4)
+        e(dv(a(.UInt8 *) (255)) babad_in5)
+        e(dv(.String) s1)
+        e(dv(.String) stringOut)
+        e(dv(.Boolean) isSame)
+    )
+    clump (
+        Copy(noError err)
+        ByteArrayToString(stringOut babad_in1 0 err)
+        Printf("1: array = <%z>, string = <%s>, err = <%z>\n" babad_in1 stringOut err)
+
+        Copy(noError err)
+        ByteArrayToString(stringOut babad_in2 0 err)
+        Printf("2: array = <%z>, string = <%s>, err = <%z>\n" babad_in2 stringOut err)
+
+        Copy(noError err)
+        ByteArrayToString(stringOut babad_in3 0 err)
+        Printf("3: array = <%z>, string = <%s>, err = <%z>\n" babad_in3 stringOut err)
+
+        Copy(noError err)
+        ByteArrayToString(stringOut babad_in4 0 err)
+        Printf("4: array = <%z>, string = <%s>, err = <%z>\n" babad_in4 stringOut err)
+
+        Copy("" stringOut)
```

Comment:
Is this line necessary? It's not used in the previous calls.

Reply:
Updated all the test cases to write default values before calling the node under test.